pkg/manifest/subscription.go

```diff
@@ -128,10 +128,21 @@ func subscriptionService(
 	}
 
 	var commands []string
+	if subscriptionOptions.ServerUrl != "" {
+		commands = append(commands, fmt.Sprintf("/usr/sbin/subscription-manager config --server.hostname %s", shutil.Quote(subscriptionOptions.ServerUrl)))
+	}
```

Comment:
Previously, both server hostname and base URL were set. These are two separate configuration options in rhsm:

```
  --server.hostname SERVER.HOSTNAME
                        Section: server, Name: hostname
...
  --server.proxy_hostname SERVER.PROXY_HOSTNAME
                        Section: server, Name: proxy_hostname
```

Now I only see hostname being set, this does not appear to be correct. We want to keep the ability to override both values, this is important for Satellite users who need to set both values.

What is good and step up from the previous version is that these default values are not hardcoded in the library now, this is good. Because typically this configuration is written by a specific Satellite-generated RPM package called `katello-ca.rpm`.